uk_gov_dash_components/status.py

```diff
@@ -5,42 +5,71 @@
 
 class status(Component):
     """A status component.
-Display ststus
+    Display ststus

```

Comment:
```suggestion
    Display status

```
spotted a typo on this!